### DISCUSSION TITLE
Inset:; property producing misaligned solutions?

### DISCUSSION THREAD

**Discussion by finlay-x** [cssbattle/forum#122] [Bugs reports]
I was using the css inset:; property to try postioning a solution for Target #24, It displays and aligns perfectly in the output preview, however only scores 96.4%. When I change the positioning to use margins I get 100%.

Also when I use https://codepen.io/alexzaworski/pen/vwmNKr to test the code that uses inset:; I get a difference of:

![Screenshot_2](https://user-images.githubusercontent.com/67915426/136453043-a16f478b-d7f4-4630-bd92-fddabaaba095.jpg)

And if I adjust the inset:; to match in the codepen previewer and paste it back into cssbattle, I get 100%, despite it completely misaligned with the cssbattle previewer

Both margin solution and inset solution are below, am I being stupid (very possible) or is this a bug with the inset:; property?

Margin solution
```
<p><p>
  <style>
    *{background:#62306D}
    p{
      background:#F7EC7D;
      width:100;
      height:150;
      margin:50 72;
      border-radius:53q;
      box-shadow:inset 0 53q#AA445F
    }
    p+p{
      margin:-150 212;
      box-shadow:inset 0-53q#E38F66
```

Inset solution
```
<p><p>
<style>
  *{background:#62306D}
  p{
    background:#F7EC7D;
    width:100;
    height:150;
    position:absolute;
    inset:50px 80px;
    border-radius:50px;
    box-shadow:inset 0 50px#AA445F
  }p+p{
    inset:84px 220px;
    box-shadow:inset 0-50px#E38F66
```

**Comment by rafehimself** [comment]
I have problems with inset shadow as well, silly machine doesn't tolerate it as valid solutions,
somehow I get 100% with slightly worse mismatch without inset shadow, but 99.9% with inset shadow, regardless of almost perfect match when clicking see the difference

**Reply by chinchang** [reply]
Hey @finlay-x you are supposedly playing on Firefox, right? Its recommended to play on Chromium based browsers only to be consistent with the scoring environment. On Chrome, for example, your inset code is mismatched.

**Reply by rafehimself** [reply]
I found shorter solution in #88 and it's even more tight as tittle suggest, but machine doesn't accept it,
![image](https://user-images.githubusercontent.com/58576132/137589212-61065d6c-dd58-477f-9f1f-309340ec98f3.png)
for my eye it's the same thing, and I have seen bigger mismatches accepted,
doesn't work in other battles as well like #88
I'm not sure should I share my code here while battle is still open 🤣 
I couldn't find any way to show you the code in PM

**Reply by chinchang** [reply]
You can DM us the code on twitter - https://twitter.com/css_battle

**Reply by rafehimself** [reply]
I don't have twitter ...
